useful_resources/sample_code/apply_activist_code.py

```diff
@@ -1,22 +1,40 @@
+### METADATA
+
+# Connectors: Redshift, VAN
+# Description: Gets activist codes stored in redshift and applies to users in Van
+
+
+### CONFIGURATION
+
+# Set the configuration variables below or set environmental variables of the same name and leave these
+# with empty strings.  We recommend using environmental variables if possible.
+
+config_vars = {
+    # Redshift  (note: this assumes a Civis Platform parameter called "REDSHIFT")
+    "REDSHIFT_PORT": "",
+    "REDSHIFT_DB": "",
+    "REDSHIFT_HOST": "",
+    "REDSHIFT_CREDENTIAL_USERNAME": "",
+    "REDSHIFT_CREDENTIAL_PASSWORD": "",
+    # Van  (note: below, we'll be dynamically reading API keys from multiline Civis credential)
+    "VAN_PASSWORD": "",
+    "VAN_DB_NAME": ""   #  FIXME: script had 'MyVoters' - is this a universal default?
+}
+
+
+### CODE
+
 from parsons import Table, Redshift, VAN
 import logging
 import os
 
-# Redshift setup - this assumes a Civis Platform parameter called "REDSHIFT"
-
-set_env_var(os.environ['REDSHIFT_PORT'])
-set_env_var(os.environ['REDSHIFT_DB'])
-set_env_var(os.environ['REDSHIFT_HOST'])
-set_env_var(os.environ['REDSHIFT_CREDENTIAL_USERNAME'])
-set_env_var(os.environ['REDSHIFT_CREDENTIAL_PASSWORD'])
-rs = Redshift()
+# Setup
 
-# AWS setup - this assumes a Civis Platform parameter called "AWS"
+for name, value in config_vars.items():    # sets variables if provided in this script
```

Comment:
This might be worth pulling out into a utility function at some point in Parsons-proper that we could build out over time for configuring the environment.